### DISCUSSION TITLE
Having a cancel button

### DISCUSSION THREAD

**Discussion by bobred** [qatrackplus/qatrackplus#633] [Ideas]
Hi would it be possible to add a cancel button to the admin pages. For instance, when editing a TestList at the bottom there are three options, Save as New, Save and continue editing and SAVE, it would be nice to have a CANCEL button that takes you back to the previous page without submitting to the database..
James

**Comment by randlet** [comment]
I'd probably accept a PR for this if someone wanted to tackle it, but I don't see a great amount of value in it myself when the browser back button has the same function.
